CMakeLists.txt

```diff
@@ -21,73 +22,118 @@ endif()
 
 find_package(ROOT REQUIRED CONFIG)
 
-# If ROOT_SOURCE_DIR is set, roottest is built as part of the root build.
-# If ROOT_SOURCE_DIR is not set, search for an installation.
-if(DEFINED ROOT_SOURCE_DIR)
-  # root-config --prefix helps to retrieve ROOTSYS env variable.
-  # ROOT_BINDIR is set by ROOTConfig.cmake
-  find_program(ROOT_CONFIG_EXECUTABLE root-config PATHS ${ROOT_BINDIR})
-  if(NOT ROOT_CONFIG_EXECUTABLE)
-    message(ERROR "root-config is not found, please rerun configuration step.")
+if(MSVC)
+
+  set(libsuffix .dll)
+  set(exesuffix .exe)
+  set(CMAKE_CXX_FLAGS "/nologo /Zc:__cplusplus /MD /GR /EHsc- /W3 /D_WIN32")
```

Comment:
It is very surprising to me that we need to specify all these flags.

Maybe we should add a comment what they do.
I think /nologo should not be needed.
/Zc:__cplusplus ties the version reported by the equally named macro to the /std:c++xx argument.
/MD should be turned on by default by CMake, so I do not understand why it is necessary.
/GR enables RTTI and CMake should turn it on by default as well.
/EHsc- disables exceptions IIRC?
/W3 is the default for the warning level, I think we should not need it.
/D_WIN32 is defined by the compiler itself, so I wonder why you specify it.

Reply:
We need the same flags than ROOT, otherwise it doesn't build correctly (ask MS)